portafly/doc/ux_specs/accounts/index.md

```diff
@@ -7,15 +7,15 @@
 * [Parent JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4724)
 
 ##### Page header
-* Create button
-  * Icon is `plus-circle`
```

Comment:
Does this mean there's no icon anymore?

Reply:
Yes, we turned the link+icon button into primary and secondary buttons to align with App Services design conventions.